lib/Geo/GDAL/FFI.pm

```diff
@@ -2045,6 +2062,16 @@ confessed if a method fails. This is the default.
 Unset the Perl function to catch GDAL errors. If no other error
 handler is set, GDAL prints the errors into stderr.
 
+=head1 NOTES ABOUT THREAD-SAFETY
+
+This module is mostly thread-safe, but for the error handling management.
```

Comment:
Could this line be deleted once the PR is in?  Or is it best left for future reference? If the latter then some rephrasing would be good.  e.g. "This module is thread safe provided the error handling is taken care of."

Reply:
Done.